### PR TITLE
{compiler}[GCC/8.3.0] Clang v8.0.1

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/627e5e26ba86beda297fba62d887ada0 for a full test report.

File: easybuild/easyconfigs/c/Clang/Clang-8.0.1-GCC-8.3.0-CUDA-10.1.243.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    '3e85dd3cad41117b7c89a41de72f2e6aa756ea7b4ef63bb10dcddf8561a7722c',  # openmp-8.0.1.src.tar.xz
+    '9fba1e94249bd7913e8a6c3aadcb308b76c8c3d83c5ce36c99c3f34d73873d88',  # lld-8.0.1.src.tar.xz
+    '7f0652c86a0307a250b5741ab6e82bb10766fb6f2b5a5602a63f30337e629b78',  # libcxx-8.0.1.src.tar.xz
+    'b75bf3c8dc506e7d950d877eefc8b6120a4651aaa110f5805308861f2cfaf6ef',  # libcxxabi-8.0.1.src.tar.xz
+    '173da6b7831a66d2f7d064f0ec3f6c56645a7f1352b709ceb9a55416fe6c93ce',  # libcxx-8.0.0-ppc64le.patch
+]
+
+dependencies = [
+    # since Clang is a compiler, binutils is a runtime dependency too
+    ('binutils', '2.32'),
```
Comment (smoors) [inline easybuild/easyconfigs/c/Clang/Clang-8.0.1-GCC-8.3.0-CUDA-10.1.243.eb:54]: @Flamefire the GCC toolchain already contains binutils, so can be removed here.

Comment (Flamefire) [reply easybuild/easyconfigs/c/Clang/Clang-8.0.1-GCC-8.3.0-CUDA-10.1.243.eb:54]: Yes but the toolchain is only used during build but binutils is a runtime dependency where GCC is not loaded. At least this is how I understand the comment above.

Comment (smoors) [reply easybuild/easyconfigs/c/Clang/Clang-8.0.1-GCC-8.3.0-CUDA-10.1.243.eb:54]: hm, ok.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node357.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/ee9d35f53f4f7c2f177f08d9df3dd3ba for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Flamefire!

**Pull request merged**